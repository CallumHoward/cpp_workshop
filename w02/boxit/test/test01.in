5
2 3 4 5
4
5
4
2 4 6 7
