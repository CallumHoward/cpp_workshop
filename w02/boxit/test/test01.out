3 4 5
60
3 4 5
60
4 6 7
